src/ansys/fluent/visualization/graphics/graphics_windows.py

```diff
@@ -168,3 +191,42 @@ def close_windows(
             graphics_windows_manager.close_windows(
                 windows_id=[self.window_id], session_id=session_id
             )
+
+
+# Define the main window
+class MainWindow(QMainWindow):
+    def __init__(self):
+        super().__init__()
+        self.setWindowTitle("Multiple PyVista Plots in Tabs")
+        screen = QApplication.primaryScreen().availableGeometry()
```

Comment:
Minor: This can have some unintended behavior for multiple screens. Ideally, the pyfluent window should be displayed in the same screen of the Fluent window, but I'm sure if it is straightforward to propagate that information from Fluent.

Reply:
ok, though I have tested this behavior in multiple screens as well without issues, we can work on this later.